### ISSUE TITLE
Update PDF14 to be in sync with PDF/UA and also ensure footer information is available in other means.

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wcag#99]
I have compared what PDF/UA and WCAG say about headers and footers and have written up a small report and recommendations on how I think we should proceed with Headers and Footers in PDF documents.

http://davidmacd.com/blog/pdf-headers-footers.html
I think we will need to update WCAG PDF14 to ensure it is clear that headers and footers will become artifacts and thtat the information needs to be available through other means (as in #1-3 below).I think PDF14 requires an update quickly. I could take an action item to propose the rewrite.
- Mark up headers and footers as artifacts as required by the PDF/UA (this is done automatically if exported from Word)
- Provide header/ footer information in a prominent place in the main body document. For example:
  - Title Page that contains author, date, copyright, classified information, form numbers etc...
  - Introduce any information that is changed in the Header/Footer in a prominent place in the main body. (e.g., proper Section Headings)
- Sync page numbers in the footers with the Page Thumbnail view using WCAG Technique PDF17 so screen reader users can determine the page they are on.

**Comment by Ryladog** [comment]
David, I think this is a very good idea. You have more initiative and energy that any accessibility-nik I know!....:-)

**Comment by DavidMacDonald** [comment]
As per discussion on the working group list. Let's do two things:
1) New failure
Failure: Information in the header or footer of a PDF is not presented at least once in the main body of the document or is not programmatically determinable.

2) Update PDF14 to clarify that a pagination artifact is ignored by screen readers, so that it does not interrupt the user.

**Comment by DavidMacDonald** [comment]
Possible wording for Kerstin's proposal:
Failure of 1.3.1 due to information in PDF headers and footers, marked as artifacts, not being programmatically available in another way.

This approach is supported by Kerstin, Jonathan, and Allen. Kerstin's proposed failure has the following qualities:

* It does not require footers to be marked up as artifacts, only that the information be available or programmatically determinable at least once in the document.
* It allows for Jonathan's recommendations regarding exposing page numbers and other changing information
* It allows for my preference of exposing header and footer information in the main body as section headings, synced page numbers etc..
* It allows for Allen's preference to have footer's inline
* It does not infringe on Gregg's WCAG2ICT concern about full PDF documents mapping to a web page
* It is oriented towards the the PDF/UA's requirement of marking them up as artifact's but is not limited by it, for things such as Jonathan's exceptions above

For pastoral direction on the best way to provide headers and footers. Technique PDF 14 can be amended to address all of our concerns because it is a technique and not a failure. It can provide the best practice, and provide example edge cases.

**Comment by DavidMacDonald** [comment]
This will be straight forward, but I'm still not set up with source tree. I hope to have it going in the next couple of weeks.

**Comment by DavidMacDonald** [comment]
I've downloaded the new Github for Desktop, forked WCAG, Synced the repository with my local version.  I created a new failure 94 in the folder source >techniques > failures 

There may be more tweaks to the failure after group discussion but I think it's on track. This is my first attempt to create a new technque and pull request it. Let me know how it turns out.

**Comment by awkawk** [comment]
David, that is in your repo, to get it into the working branch in the official WCAG repo you need to do a pull request from your repo to the right repo here (which we need to make because we are in that odd time when we are between approval to publish for public review and actually doing it.  We will let you know in the next couple days what repo to target.

**Comment by awkawk** [comment]
Ah, I see that you did this already.  We will just redirect the pull (which is into the working branch rather than the master, thank you) when we are ready to accept it (following WG discussion)

**Comment by joshueoconnor** [comment]
Surveyed Jan 5th https://www.w3.org/2002/09/wbs/35422/20161stSurvey/results

**Comment by alastc** [comment]
Hi @DavidMacDonald, I'm reviewing old issues, it looked like this got closed without actually actioning, but I can't see the changes, do you know if this is still valid?
